src/ZoneServer/Skills/Handlers/Scout/ObliqueFire.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Drawing;
+using System.Linq;
+using Melia.Shared.L10N;
+using Melia.Shared.Tos.Const;
+using Melia.Shared.World;
+using Melia.Zone.Network;
+using Melia.Zone.Skills.Combat;
+using Melia.Zone.Skills.Handlers.Base;
+using Melia.Zone.Skills.SplashAreas;
+using Melia.Zone.World.Actors;
+using Melia.Zone.World.Actors.CombatEntities.Components;
+using Yggdrasil.Extensions;
+using static Melia.Zone.Skills.SkillUseFunctions;
+
+namespace Melia.Zone.Skills.Handlers.Scout
+{
+	/// <summary>
+	/// Handles the Scout skill Oblique Fire.
+	/// </summary>
+	[SkillHandler(SkillId.Scout_ObliqueFire)]
+	public class ObliqueFire : ITargetSkillHandler
+	{
+		/// <summary>
+		/// Handles the skill, shoot missile at enemy that spreads to another target.
+		/// </summary>
+		public void Handle(Skill skill, ICombatEntity caster, ICombatEntity designatedTarget)
+		{
+			if (!caster.TrySpendSp(skill))
+			{
+				caster.ServerMessage(Localization.Get("Not enough SP."));
+				return;
+			}
+
+			skill.IncreaseOverheat();
+			caster.TurnTowards(designatedTarget);
+			caster.Components.Get<CombatComponent>().SetAttackState(true);
+
+			if (designatedTarget == null)
+			{
+				Send.ZC_SKILL_FORCE_TARGET(caster, null, skill, null);
+				return;
+			}
+
+			if (!caster.Position.InRange2D(designatedTarget.Position, skill.Data.MaxRange))
+			{
+				caster.ServerMessage(Localization.Get("Too far away."));
+				return;
+			}
+
+			var damageDelay = TimeSpan.FromMilliseconds(200);
+			var skillHitDelay = skill.Properties.HitDelay;
+
+			var skillHits = new List<SkillHitInfo>();
+
+			var skillHitResult = SCR_SkillHit(caster, designatedTarget, skill);
+			designatedTarget.TakeDamage(skillHitResult.Damage, caster);
+
+			var skillHit = new SkillHitInfo(caster, designatedTarget, skill, skillHitResult, damageDelay, skillHitDelay);
+			skillHit.ForceId = ForceId.GetNew();
+
+			skillHits.Add(skillHit);
+
+			var originPos = caster.Position;
+			var hits = new List<HitInfo>();
+			var splashArea = new Circle(designatedTarget.Position, skill.Data.SplashHeight * 2);
```

Comment:
I feel like we shouldn't make arbitrary calculations on data values to make the data work, as it seems unlikely that that's the intended way. If they had wanted the radius to be that amount, they probably would've made SplashHeight twice as big. It's not pretty, but as long as we don't have a reliable source for these values we should rather use raw numbers. That way it seems less like this is presumed to be official behavior.

Reply:
I saw that the SlashHeight was being used to calculate the circle radius in the source files.
But in the case of this skill it, the range doesn't seem to match the range in the official server.
Multiplying by 2 give me a good satisfied result.

Reply:
Right. That's totally fine, but this way it reads like this is intended, official behavior, but that seems rather unlikely and could lead to issues or confusion down the line. That's why it would be better to hardcode the value and potentially make a note of it instead of doing calculations on arbitrary data that happened to contain values that somewhat worked.